### ISSUE TITLE
More reminders about region expiring

### ISSUE THREAD

**Issue by Bloodlex** [alex9849/advanced-region-market#133]
Hey,

Could you please add more notifications about region expiring during the game? A lot of my players complain that they don't see the notification upon entering the server. Perhaps some kind of title in the middle of the screen every X minutes or something similar.

Thanks for your help.

Best regards,
Bloodlex

**Comment by Roos-Skywalker** [comment]
That would be extremely helpful. If the rented plot has 24 hours left, simply notify the online player every hour.

**Comment by Bloodlex** [comment]
Yes, please make it more configurable, so we can spam our players with notifications 🙂. They want it in order not to lose the access to their plots.

**Comment by SiuanSong** [comment]
I'd also like to suggest the ability to send custom commands for expirations with multiple custom times.  Being able to send a /mail 3 times before a plot expires would be incredibly helpful.

Something like this:

RentRegionExpirationWarningTimes: 
  Expiration1:
    - Time: 7d
    - Command: /mail send Your Rent Region will expire in 7 days.
    - SendAdditionalMessageinChat: True
  Expiration2:
    - Time: 3d
    - Command: /mail send Your Rent Region will expire in 3 days.
    - SendAdditionalMessageinChat: True
   Expiration3:
    - Time: 1d
    - Command: /mail send Your Rent Region will expire in 1 day.
    - SendAdditionalMessageinChat: True
